core/camera/src/main/java/com/google/jetpackcamera/core/camera/CameraSession.kt

```diff
@@ -615,13 +615,11 @@ private suspend fun startVideoRecordingInternal(
                             )
                         )
                     }
-
                     ERROR_DURATION_LIMIT_REACHED -> {
                         currentCameraState.update { old ->
                             old.copy(
                                 videoRecordingState = VideoRecordingState.Inactive(
-                                    // cleanly display the max duration
-                                    finalElapsedTimeNanos = maxDurationMillis * 1_000_000
+                                    finalElapsedTimeNanos = maxDurationMillis * 1_000_000_000
```

Comment:
Wouldn't milliseconds * 1,000,000,000 be picoseconds?

If we want to be more explicit about this conversion, we can do `maxDurationMillis.milliseconds.inWholeNanoseconds`.

Reply:
oops, that looks like an accidental change. 
Thanks for the conversion snippet :)